src/applications/disability-benefits/all-claims/pages/individualsInvolvedFollowUp.js

```diff
@@ -5,7 +5,6 @@ import {
   personDescriptionText,
 } from '../content/individualsInvolved';
 import IndividualsInvolvedCard from '../components/IndividualsInvolvedCard';
-import fullNameUI from 'platform/forms/definitions/fullName';
```

Comment:
I'm not sure why this was removed? It appears to still be in use in the `covidResearchUISchema` above.

Reply:
Long slack thread here: https://dsva.slack.com/archives/CBU0KDSB1/p1598458760323500


TL;DR: individualsInvolvedFollowUp.js was using `fullNameUI` (which includes suffix) for the UISchema but a custom element for Schema.  This has not been an issue until the unit tests for `coronavirus-screener` were introduced with this PR.  For some reason, the tests in [this file ](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/disability-benefits/all-claims/tests/pages/individualsInvolvedFollowUp.unit.spec.jsx) began failing due to error: `Error: uiSchema order list contains extraneous property 'suffix'.`.  I removed `fullNameUI` and replaced with a custom element in the UISchema to see if all test would pass in Circle. 

I plan to seek out an engineer more familiar with `individualsInvolvedFollowUp.js` and ask them to review this as well.  

@mchelen-gov and @billfienberg  are also helping analyze this.